rpcs3/rpcs3qt/rsx_debugger.cpp

```diff
@@ -577,7 +573,8 @@ void rsx_debugger::OnClickDrawCalls()
 		if (width && height && !draw_call.depth_stencil[1].empty())
 		{
 			const std::span<const std::byte> orig_buffer = draw_call.depth_stencil[1];
-			u8* buffer = static_cast<u8*>(std::malloc(4ULL * width * height));
```

Comment:
How'd this ever get merged? :eyes:

Reply:
probably as part of the first Qt or wx pr xD

Reply:
The whole function is dead code I'm pretty sure.